<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardioDialog</class>
 <widget class="QWidget" name="CardioDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="paddelErgometerCanuLabel">
       <property name="text">
        <string>paddelErgometerCanu</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="paddelErgometerCanuLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="paddelErgometerKayakLabel">
       <property name="text">
        <string>paddelErgometerKayak</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="runningLabel">
       <property name="text">
        <string>running</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="byciclingLabel">
       <property name="text">
        <string>bycicling</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="paddelErgometerKayakLineEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="runningLineEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="byciclingLineEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="visibilityLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="visibilityLabel">
       <property name="text">
        <string>visible</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Speichern</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delButton">
       <property name="text">
        <string>LÃ¶schen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
